packages/docs-page/index.tsx

```diff
@@ -9,6 +9,7 @@ import HashiHead from '@hashicorp/react-head'
 import { MDXRemote, MDXRemoteSerializeResult } from 'next-mdx-remote'
 import { SearchProvider } from '@hashicorp/react-search'
 import { AlgoliaConfigObject } from '@hashicorp/react-search/types'
+import type { Products } from '@hashicorp/platform-product-meta'
```

Comment:
Can you add `@hashicorp/platform-product-meta` to the `package.json` for `docs-page`?

Reply:
Done!